python/cugraph/structure/graph.py

```diff
@@ -184,56 +195,60 @@ def from_cudf_edgelist(self, input_df, source='source',
-                raise Exception('edge_attr should be a list of column names')
+                raise Exception("edge_attr should be a list of column names")
             value_col = {}
             for col_name in edge_attr:
                 value_col[col_name] = elist[col_name]
         elif edge_attr is not None:
             value_col = elist[edge_attr]
         else:
             value_col = None
-        renumber_map = None
-        if renumber:
-            if type(source) is list and type(destination) is list:
-                source_col, dest_col, renumber_map = multi_rnb(elist,
-                                                               source,
-                                                               destination)
-            else:
-                source_col, dest_col, renumber_map = rnb(elist[source],
-                                                         elist[destination])
-            self.renumbered = True
-        else:
-            if type(source) is list and type(destination) is list:
-                raise Exception('set renumber to True for multi column ids')
-            else:
-                source_col = elist[source]
-                dest_col = elist[destination]
+
         if not self.symmetrized and not self.multi:
             if value_col is not None:
-                source_col, dest_col, value_col = symmetrize(source_col,
-                                                             dest_col,
-                                                             value_col)
+                source_col, dest_col, value_col = symmetrize(
+                    source_col, dest_col, value_col
+                )
             else:
                 source_col, dest_col = symmetrize(source_col, dest_col)
 
-        self.edgelist = Graph.EdgeList(source_col, dest_col, value_col,
-                                       renumber_map)
+        self.edgelist = Graph.EdgeList(
+            source_col, dest_col, value_col
+        )
+        self.renumber_map = renumber_map
 
     def add_edge_list(self, source, destination, value=None):
-        warnings.warn('add_edge_list will be deprecated in next release.\
- Use from_cudf_edgelist instead')
+        warnings.warn(
+            "add_edge_list will be deprecated in next release.\
+ Use from_cudf_edgelist instead"
+        )
         input_df = cudf.DataFrame()
-        input_df['source'] = source
-        input_df['destination'] = destination
+        input_df["source"] = source
+        input_df["destination"] = destination
         if value is not None:
-            input_df['weights'] = value
-            self.from_cudf_edgelist(input_df, edge_attr='weights')
+            input_df["weights"] = value
+            self.from_cudf_edgelist(input_df, edge_attr="weights")
         else:
             self.from_cudf_edgelist(input_df)
 
-    def from_dask_cudf_edgelist(self, input_ddf):
+    def from_dask_cudf_edgelist(self, input_ddf, renumber=False):
```

Comment:
`renumber=False` should be True to be consistent with the single GPU `from_cudf_edgelist`.
Notice that later degree and pagerank MG tests use both in the same test without specifying this option which would result in different input. I think these tests pass just by luck because the input is already renumbered.

Reply:
Agreed.  I set it as False so that I would - by default - not change the existing MG unit tests.

Reply:
:+1: 

One more thing I realized is that the doc regarding renumbering is outdated for 
- `from_cudf_edgelist`
- `from_dask_cudf_edgelist`

Reply:
Updated this to True now that everything is working correctly.

I copied the renumber parameter documentation over to the from_dask_cudf_edgelist in the latest push.